server/util/personas.js

```diff
@@ -2,8 +2,9 @@
 
  const personaA = {
     key: 'a',
-    name: 'Alice Adams',
-    face: 'face9.jpeg'
+    name: 'Emeka Eze',
```

Comment:
Why did you rename personA to have the same name as personE? 

I assume this was accidental?

Reply:
yes im reverting all file from the PR except people.js